inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/DynamicWorkloadManagementPage.java

```diff
@@ -150,12 +150,12 @@ public DynamicWorkloadManagementPage(final PageParameters aPageParameters)
         //Columns of the table
         //Each column creates TableMetaData
         List<IColumn<SourceDocument, String>> columns = new ArrayList<>();
-        columns.add(new LambdaColumn<>(new ResourceModel("Document"), SourceDocument::getName));
-        columns.add(new LambdaColumn<>(new ResourceModel("Finished"), _doc -> dataProvider
+        columns.add(new LambdaColumn<>(new ResourceModel("Document"), getString("Document"), SourceDocument::getName));
+        columns.add(new LambdaColumn<>(new ResourceModel("Finished"), getString("Finished"), _doc -> dataProvider
                 .getFinishedAmountForDocument((SourceDocument) _doc)));
-        columns.add(new LambdaColumn<>(new ResourceModel("Processed"), _doc -> dataProvider
+        columns.add(new LambdaColumn<>(new ResourceModel("Processed"), getString("Processed"), _doc -> dataProvider
                 .getInProgressAmountForDocument((SourceDocument) _doc)));
-        columns.add(new LambdaColumn<>(new ResourceModel("Users"), _doc -> dataProvider
+        columns.add(new LambdaColumn<>(new ResourceModel("Users"), getString("Users"), _doc -> dataProvider
```

Comment:
Why do you add the `getString()` here again?

Reply:
Otherwise the columns are not sortable and initially we wanted them to be sortable

Reply:
Ok. But by `getString()` - this method is used to obtain an internationalized string for a given resource bundle key. Why does the sort property depend on the language of the UI? That said, `getString()` must not be used by code which is called from the constructor.

Reply:
I used getString() here, because the sorting in "DataProvider" depends on the column headers strings. And the headers are all internationalzed strings. But if getString() must not be called here, I can also change that by using headers(0), headers(1) ... and so on if you prefer

Reply:
As it stands, that won't work either since `headers` is also defined in terms of `getString()`:

```
        List<String> headers = new ArrayList<>();
        headers.add(getString("Document"));
```

One option could be that you create an enum which defines the columns of the table. Btw: `DataProvider` is a very generic name. Would it be possible to use a slightly more self-explanatory name? What data does it provide?

Reply:
Enum would be a nice solution, sure I'll change it to that.
Ok how about "TableContentProvider"?

Reply:
That still doesn't say what it provides. It sounds like a generic re-usable class - or rather like a high-level interface. But is that what it is? Maybe something like `AnnotationQueueOverviewDataProvider`(given... the "queue" isn't quite there yet).